### PR TITLE
Rework fire axe ferals into something more dangerous.

### PR DESCRIPTION
**Pull request by Aerin-of-the-Toast**
#### Summary
Features "Replace axe wielding feral humans with deranged axemen"

#### Purpose of change

Feral humans wielding fire axes aren't very dangerous for swinging around, ya know, an entire fire axe. While stronger than the other two basic ferals, their damage still seemed on the low end and they mostly felt like a fire axe delivery service rather than a true threat.

#### Describe the solution

Fire axe ferals have had their damage buffed from 2d7 bash + 9 cut (an average of 8 bash and 9 cut for 17 total damage) to 3d7 bash + 12 cut (an average of 12 bash and 12 cut for 24 total damage), using the battle axe-wielding Armored Feral (3d7+15) as a point of reference. Changed their name from just "feral human" to "deranged axeman" with an accompanying description change due to their newly increased danger level. Removed their ability to throw rocks under the assumption they're too busy holding their axe in both hands. 

#### Describe alternatives you've considered

Increasing the damage even further with the idea that getting hit with an entire axe should nearly kill you on the spot, but that seemed far too sadistic. I also toyed with giving them more melee skill or health to make them more fearsome combatants rather than just a basic feral human with a dangerous weapon, but I figured the latter would be fine enough. The main thing threatening about them is the axe, not that they're an expert with it or extremely athletic. I also toyed with the idea of making them feral firefighters, which would give them some armor from the bunker gear as well, but decided that could be a separate mob.

#### Testing

Spawned one in, let it take a few swings at me, noticed that my health was getting chunked rather hard as appropriate.

#### Additional context

I'm hoping this makes axe ferals serve more as an early "stop trying to melee everything and get a gun you fool" wake-up call, as they're just as easy to kill with literally any ranged weapon as before. Their lack of ranged attack means they're arguably easier to kite with a spear than before though, but now it's much more dangerous if you mess up. This level of melee damage also means they're still threatening to players in armor too, unless they go full high-tempered 6mm thick plate I suppose.

### CODE REVIEW

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -160,13 +160,38 @@
   {
     "id": "mon_feral_human_axe",
     "type": "MONSTER",
-    "description": "Their pupils are dilated and what can be seen of the iris and sclera are bloodshot.  This axe-wielding maniac still breathes, but the zombies treat them like one of their own.",
-    "copy-from": "mon_feral_human_pipe",
-    "melee_dice": 2,
+    "name": { "str": "deranged axeman", "str_pl": "deranged axemen" },
+    "description": "They stumble along dragging a bloodstained fire axe.  Their face is twisted into a sadistic grin, and they walk among the dead as though already one of them.",
```
Comment (TheShadowFerret) [inline data/json/monsters/feral_humans.json:164]: ``fire axe``
*angry #58014 noises*

Comment (Aerin-of-the-Toast) [reply data/json/monsters/feral_humans.json:164]: Oops, didn't notice that PR. Easy fix at least, I'll just change it so the type of axe is unspecified.

**Comment by fairyarmadillo** [comment]
Your damage values may be off a bit as there's a PR in to add a lot of different axes to their drop pool, some of which are weaker than the fire axe. [#58014](https://github.com/CleverRaven/Cataclysm-DDA/pull/58014) 

Disregard if you've accounted for this.

**Comment by Aerin-of-the-Toast** [comment]
> Your damage values may be off a bit as there's a PR in to add a lot of different axes to their drop pool, some of which are weaker than the fire axe. [#58014](https://github.com/CleverRaven/Cataclysm-DDA/pull/58014) 
> 
> Disregard if you've accounted for this.
> 
> These enemies are all over the place on day one. Increasing their lethality may justify making them rarer, there's also an argument to be made for doing that based on the fact that there aren't axes laying around all over the place in real life, so the crowbar and especially pipe guys should be more common.

They're already a fair bit rarer than the other two kinds of ferals. And while there might not theoretically be a lot of axes "lying around," way I see it since ferals have a whole city to bumble around in without any danger (since zombies ignore them) it wouldn't be too hard to find one axe or hatchet somewhere, like from a sporting goods store or the like. And presumably they'd be drawn to an axe as being more dangerous than most other things they can find. 

Regarding damage values, since there isn't a way to make some axe ferals attack faster with weaker attacks than others without making a whole separate mob with its own drop tables, I think a feral with a hatchet or crash axe and a feral with a wood axe or fire axe are all roughly as dangerous as each other anyway overall. Maybe later they could be split into two mobs, one using/dropping the two-handed axes and one dropping/using one-handed axes. That actually gives me an idea, but later :p

**Pull request merged**